### ISSUE TITLE
ORKKeychainWrapper - NSSecureCoding Swift Syntax

### ISSUE THREAD

**Issue by swaffoja** [ResearchKit/ResearchKit#797]
TL;DR... How can I retrieve a string from the Keychain using ORKKeychainWrapper.objectForKey() in Swift?

I'd like to use `ORKKeychainWrapper` to store and retrieve `String` values in the Keychain. I'm having trouble understanding the correct syntax to use with `NSSecureCoding` in Swift. I created a class with three methods:

    // Example set, get, & remove
    KeychainManager.set("Foo", forKey: "DESIRED-KEY")
    let myString: String? = KeychainManager.get("DESIRED-KEY")
    KeychainManager.remove("DESIRED-KEY")

Unfortunately, the `if let value = ORKKeychainWrapper.objectForKey(key, error: &error) as? String ` line in the following class generates an `EXC_BAD_ACCESS` error when the key does not exist. **How do I properly retrieve a string using** `ORKKeychainWrapper.objectForKey()` **in Swift?**

    class KeychainManager {

        static func get(key: String) -> String? {

            var error: NSError?
            if let value = ORKKeychainWrapper.objectForKey(key, error: &error) as? String {
                return value
            }

            if error != nil {
                print(error)
            }

            return nil
        }

        static func set(value: String, forKey key: String) {

            do {
                try ORKKeychainWrapper.setObject(value, forKey: key)            
            } catch let error {
                print(error)
            }

        }

        static func remove(key: String) {

            var error: NSError?
            ORKKeychainWrapper.objectForKey(key, error: &error)

            if (error == nil) {
                do {
                   try ORKKeychainWrapper.removeObjectForKey(key)
                } catch let removeError {
                    print(removeError)
                }
            } else {
                print(error)
            }

        }
    }

**Comment by swaffoja** [comment]
`String` does not conform to the `NSSecureCoding` protocol. You must store and retrieve as `NSString`.
